elements/lisk-chain/src/block_header.ts

```diff
@@ -84,7 +84,7 @@ interface BlockHeaderAttrs {
 	readonly height: number;
 	readonly generatorAddress: Buffer;
 	readonly previousBlockID: Buffer;
-	readonly timestamp?: number;
+	readonly timestamp: number;
```

Comment:
I kept it optional earlier because someone can start building block header in multiple steps and want to set the time stamp in the end right before signing it.